compat/compat.c

```diff
@@ -78,6 +78,7 @@ os_strerror(int errnum, char *buf, size_t buflen)
                         WCHAR *s = gai_strerror(errno);
                         if (!WideCharToMultiByte(CP_UTF8, 0, s, -1, buf, buflen, NULL, NULL))
                                 return (-1);
+						break;
```

Comment:
Hi, you put tabs instead of spaces.

Reply:
Thanks, for my defense that patch wasn't made from my usual editor :) fixed